docs/EN_US/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml

```diff
@@ -530,6 +530,18 @@
               <entry>Upper limit of a CSV line read in MB.</entry>
             </row>
 
+            <row>
+              <entry><emphasis>validateFileType</emphasis></entry>
+
+              <entry>Default true</entry>
+
+              <entry>If false, the engines use the definition in the ECL
+              workunit and ignores the file type in the DFU record. This check
```

Comment:
>in the DFU record.

that may be misleading, it's not part of the record definition.
Perhaps change to:

"ignores the file type from the logical file meta data".